### PR TITLE
updated tg link to support link

### PR DESCRIPTION
**Pull request by prayagd**
- Update the TG link to support link as all users are not expected to TG

### CODE REVIEW

File: src/config/index.ts [hunk 1]
```diff
@@ -62,5 +62,5 @@ export const config = {
     mockSep24: false,
     overwriteMinimumTransferAmount: false,
   },
-  telegramUrl: 'https://t.me/vortex_fi',
+  telegramUrl: 'https://forms.gle/bgH4XTTbQ3YbwQ3t7',
```
Comment (ebma) [inline src/config/index.ts:65]: Nit: let's rename this to `supportUrl` then, if this is not related to telegram anymore.
```suggestion
  supportUrl: 'https://forms.gle/bgH4XTTbQ3YbwQ3t7',
```

File: src/pages/failure/index.tsx [hunk 2]
```diff
@@ -52,9 +52,9 @@ export const FailurePage = ({ finishOfframping, continueFailedFlow, transactionI
           </p>
         )}
         <p className="mb-6 text-center text-gray-400">
-          Contact our support team on{' '}
+          Contact our{' '}
           <a href={config.telegramUrl} target="_blank" rel="noreferrer" className="underline">
```
Comment (ebma) [inline src/pages/failure/index.tsx:56]:
```suggestion
          <a href={config.supportUrl} target="_blank" rel="noreferrer" className="underline">
```

**Comment by prayagd** [comment]
@ebma suggestions committed

File: src/config/index.ts (hunk 1 above)
Comment (TorstenStueber) [reply src/config/index.ts]: Good point, it is not even telegram now anymore but a Google form.
